bin/device-install.bat

```diff
@@ -49,8 +52,14 @@ IF EXIST %FILENAME% IF x%FILENAME:update=%==x%FILENAME% (
     ) else (
         %ESPTOOL_CMD% --baud 115200 write_flash 0x260000 bleota-s3.bin
     )
-    for %%f in (littlefs-*.bin) do (
-        %ESPTOOL_CMD% --baud 115200 write_flash 0x300000 %%f
+    IF %WEB_APP%==1 (
+        for %%f in (littlefsweb-*.bin) do (
```

Comment:
Shouldn't this be `littlefswebui-`? Same for the .sh script.

Reply:
You are right, though I did changed it.

Reply:
It should be correct now, is a new pr needed?

Reply:
Np